netci.groovy

```diff
@@ -169,13 +169,6 @@ class Constants {
     // Valid PR trigger combinations.
     def static prTriggeredValidInnerLoopCombos = [
         'Windows_NT': [
-            'x64': [
-                'Checked'
-            ], 
-            'x86': [
-                'Checked',
-                'Release'
-            ], 
             'arm': [
```

Comment:
I think you can remove Windows_NT arm Debug - we run this job in AzDO

Reply:
Will do.

Reply:
And same for  Windows_NT arm64 Debug